content/docs/guides/workflow-grammar.md

```diff
@@ -364,10 +371,21 @@ vars_iter: - range(0, 4)
 <pre>
 vars_iter: - [0, 1, 2, 3]
 </pre>
+
+</pre>
+In case if we need to decide this task dynamically based on the output of other task,
```

Comment:
In the case if we need to pass the inputs dynamically based on the output of other task,

Reply:
changed